stubs/redis/redis/cluster.pyi

```diff
@@ -24,8 +24,9 @@ READ_COMMANDS: Any
 
 def cleanup_kwargs(**kwargs): ...
 
-class ClusterParser(DefaultParser):
-    EXCEPTION_CLASSES: Any
+# It uses `DefaultParser` in real life, but it is a dynamic base class.
+class ClusterParser(BaseParser):
+    EXCEPTION_CLASSES: ClassVar[Any]
```

Comment:
`EXCEPTION_CLASSES` has the same type as in the superclass; I don't think we need to override it at all here

```suggestion
class ClusterParser(BaseParser): ...
```